spec/decorators/identity_decorator_spec.rb

```diff
@@ -1,16 +1,41 @@
 require 'rails_helper'
 
 describe IdentityDecorator do
+  include ActionView::Helpers::TagHelper
+
   let(:user) { create(:user) }
-  let(:identity) { create(:identity, :active, user: user) }
+  let(:service_provider) { 'http://localhost:3000' }
+  let(:identity) { create(:identity, :active, user: user, service_provider: service_provider) }
 
   subject { IdentityDecorator.new(identity) }
 
   describe '#pretty_event_type' do
-    it 'returns the localized text corresponding to the identity event' do
-      expect(subject.pretty_event_type).to eq(
-        t('event_types.authenticated_at', service_provider: identity.display_name)
-      )
+    context 'with an SP that has a return url' do
+      it 'returns the localized text with a link corresponding to the identity event' do
+        event_output = subject.pretty_event_type
+        service_provider_link = content_tag(
```

Comment:
If we're switching the content_tag above to link_to, should this one change too?